deploy.sh

```diff
@@ -19,6 +19,7 @@ helm upgrade --install --timeout 900s \
   --namespace "$AIRFLOW_NAMESPACE_PREFIX-$ENVIRONMENT" \
   -f ./helm/cas-airflow/values.yaml \
   --set-file airflow.webserver.webserverConfig=./helm/cas-airflow/webserver_config.py \
+  --set-file airflow.apiServer.apiServerConfig=./helm/cas-airflow/webserver_config.py \
```

Comment:
Just checking if this shouldn't be `apiserver_config.py` instead! I know nothing about the setup

Reply:
Confusingly enough, it gets loaded into _airflow.apiServer.apiServerConfig_ as `webserver_config.py`, as noted in the [parameters reference docs](https://airflow.apache.org/docs/helm-chart/stable/parameters-ref.html#common) (check the `apiServer.apiServerConfig` description).